### DISCUSSION TITLE
Monkeytype In Education

### DISCUSSION THREAD

**Discussion by unrealapex** [monkeytypegame/monkeytype#1555] [Ideas] [answered]
@Miodec I was wondering what you think of the idea of getting educators to use Monkeytype to teach users(children) how to type (i.e having lessons on the site in a "Lessons" section, etc...). I think the usage of Monkeytype in the education sector alongside the typist community would provide several opportunities for you and the project. What do you think?

**Comment by Miodec** [comment, marked as answer]
I dont know. It adds loads of extra work of coming up with lessons, writing texts for them, probably new graphics that include the hands and the keyboard. I feel like there are more important things to do
